docs/_quarto.yml

```diff
@@ -23,6 +25,11 @@ website:
         aria-label: GitHub
 
   sidebar:
+    - title: "Model concept"
+      contents:
+        - modelconcept/index.qmd
+        - core/equations.qmd
+        - core/allocation.qmd
```

Comment:
I agree that I need to sit together with @SnippenE to agree on the structure/ToC. For the moment I leave it here, such that this issue can be closed and a new one can be opened for further revisions.

Reply:
Agree that it is best to discuss structure separately and get the new content in. Then I'd say to just move `modelconcept/index.qmd` to `core/modelconcept.qmd` and not introduce a new top bar entry for just one page (since the equations and allocation pages shouldn't be copied there).

Reply:
done